Chap_Introduction.tex

```diff
@@ -7,162 +7,156 @@ \chapter{Introduction}
 \end{itemize}
 
-% ISSUE175: Consider presenting the full goals of the PMI efforts rather than presenting (as is currently done) PMIx as an add-on to PMI-previous.
-
-% ISSUE175: Suggestion:  “PMIX is designed to support clusters of large sizes….” and then delete all text prior to this in the chapter?
-
-\ac{PMIx} represents an attempt to resolve these questions by providing an extended version of the \ac{PMI} definitions specifically designed to support clusters up to exascale and larger sizes.
-The overall objective of the project is not to branch the existing definitions -- in fact, \ac{PMIx} fully supports both of the existing PMI-1 and PMI-2 \acp{API} -- but rather to:
-
-% ISSUE175: Need to determine if PMI-1, PMI-2 API’s should be part of the standard (documented somehow?).  Or it is sufficient to say that these can easily be implemented on top of PMIx API’s?
+\deleted{ \ac{PMIx} represents an attempt to resolve these questions by providing an extended version of the \ac{PMI} definitions specifically designed to support clusters up to exascale and larger sizes.} 
+\replaced{\ac{PMIx} was intended to expand on the PMI effort,
+and was not designed to be a radical shift away from the basic concepts introduced by
+its predecessors.  \ac{PMIx} API's were designed to easily allow a \ac{PMIx} implementation to
+be capable of easily supporting both of the existing PMI-1 and PMI-2 \acp{API} with
+minimal effort.  Since its introduction, \ac{PMIx} has expanded 
+on earlier \ac{PMI} efforts in the following ways:}{ The overall objective of the project is not to branch the existing definitions -- in fact, \ac{PMIx} fully supports both of the existing PMI-1 and PMI-2 \acp{API} -- but rather to:}
 
 \begin{compactalphaenum}
-\item add flexibility to the existing \acp{API} by adding an array of key-value ``attribute'' pairs to each \ac{API} signature that allows implementers to customize the behavior of the \ac{API} as future needs emerge without having to alter or create new variants of it;
-
-\item add new APIs that provide extended capabilities such as asynchronous event notification plus dynamic resource allocation and management;
-
-\item establish a collaboration between \ac{SMS} subsystem providers including resource manager, fabric, file system, and programming library developers to define integration points between the various subsystems as well as agreed upon definitions for associated \acp{API}, attribute names, and data types;
-
-\item form a standards-like body for the definitions; and
-
-% ISSUE175: delete?
-
-\item provide a reference implementation of the \ac{PMIx} standard.
+\item \added{\ac{PMIx} provides an extended version of the \ac{PMI} definitions specifically designed for better scalability.}
```

Comment:
Just as a heads-up: Bill Gropp took exception with a similar comment in a prior draft article. His point was quite correct - the APIs have no impact on scalability. It is the implementation which controls that behavior. I would advise perhaps leaving out references to the prior PMI definitions as all we have to do here is describe PMIx, not how it relates to other things.

Reply:
Yes, that was my reaction as well.   I suppose one might come up with an API that was especially difficult to implement efficiently, but we can't really say the API's themselves are scalable.